packages/catalog-process/src/services/validators.ts

```diff
@@ -314,3 +314,13 @@ export function assertConsistentDailyCalls({
     throw inconsistentDailyCalls();
   }
 }
+
+export function assertDescriptorQuotasUpdatable(descriptor: Descriptor): void {
```

Comment:
We already have an `isDescriptorUpdatable` that you can reuse - I would also reuse it in the corresponding method to update a descriptor which is not a template instance (https://github.com/pagopa/interop-be-monorepo/blob/f2e0cf5e63b3cb2d877b3925d4829343fe2e539f/packages/catalog-process/src/services/catalogService.ts#L1905-L1906)

Reply:
Makes sense... there are other places where it could be reused as an assertion probably, but let's not add unrelated stuff to this task, looks good to me 🙏